src/main/java/seedu/address/AppParameters.java

```diff
@@ -45,20 +45,7 @@ public static AppParameters parse(Application.Parameters parameters) {
 
     @Override
     public boolean equals(Object other) {
-        if (other == this) {
-            return true;
-        }
-
-        if (!(other instanceof AppParameters)) {
-            return false;
-        }
-
         AppParameters otherAppParameters = (AppParameters) other;
         return Objects.equals(getConfigPath(), otherAppParameters.getConfigPath());
     }
-
-    @Override
-    public int hashCode() {
-        return configPath.hashCode();
-    }
```

Comment:
wait why is this needed to be removed. isnt this just to make sure an app parameter object is the same as another app parameter with the same stuff?